Sources/StreamChatUI/ChatMessageList/ChatMessageListVCDataSource.swift

```diff
@@ -36,3 +39,9 @@ public protocol ChatMessageListVCDataSource: AnyObject {
         messageLayoutOptionsAt indexPath: IndexPath
     ) -> ChatMessageLayoutOptions
 }
+
+public extension ChatMessageListVCDataSource {
+    var isFirstPageLoaded: Bool {
+        true
```

Comment:
Why is this straight true?

Reply:
By default, if the bidirectional scrolling is not implemented (ex: Thread) View, the first page is always loaded (since there's no way to jump to mid-page).

We also needed to have default implementations for these new protocol requirements, otherwise we would break customers who are using the Message List directly